std_msgs/Bool left
std_msgs/Bool middle
std_msgs/Bool right
